src/pyedb/dotnet/edb_core/edb_data/hfss_simulation_setup_data.py

```diff
@@ -977,19 +977,14 @@ def via_mesh_plating(self):
         -------
         bool
         """
-        if self._parent._pedb.version[0] >= 10:
-            return self._via_settings.ViaMeshPlating
-        else:
-            self._parent._pedb.logger.error("Property only supported on Ansys release 2024R1 and later")
-            return False
+        return self._via_settings.ViaMeshPlating
+        self._parent._pedb.logger.warning("Property only supported on Ansys release 2024R1 and later")
```

Comment:
Just so you know, this log warning will never be triggered as a return if performed just before.